docs/otel/sampling.md

```diff
@@ -0,0 +1,101 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Sampling
+--->
+
+# Semantic Conventions for Sampling
+
+**Status**: [Experimental][DocumentStatus]
+
+<!-- toc -->
+<!-- tocstop -->
+
+These attributes reflect the effect of sampling in a telemetry
```

Comment:
which component is expected to follow this semconv and set these attributes? A sampler? User application which wants to customize sampler behavior? 


also, who's the consumer is this data? other pipeline components? samplers? backends? end users?